projects/storefrontapp/src/app/app.module.ts

```diff
@@ -31,7 +32,58 @@ if (!environment.production) {
   imports: [
     BrowserModule.withServerTransition({ appId: 'spartacus-app' }),
     BrowserTransferStateModule,
+    CdsModule,
```

Comment:
Again, after the `feature/COL-168-cms-component` is merged, this should be `CdsModule.forRoot()`